### PR TITLE
Completed css-brochure-form

### PR DESCRIPTION
**Pull request by ayshahussain141**
css-page

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  border-color: #b5b9b8;
+  padding-top: 10px;
+  padding-right: 300px;
+  padding-bottom: 9px;
+  /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+  padding-left: 10px;
+  margin-bottom: 17px;
+  border-radius: 7px;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: AtlasGrotesk;
```
Comment (Zhiwutian) [inline css-brochure-form/styles.css:36]: make sure you include a backup font for all fonts you use.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
there are two changes I would make for this one: 
1. Make sure all of your fonts have a backup font assigned to them
2.  I would wrap the title and form in a main element, then set a width on that element of around 600px, then I would add css to that main element of margin: 0 auto which takes the available margin and divides it evenly around the element that should center the main container on the screen. Its much better than using margin to position the elements away from the left of the screen. 
Great work so far!

**Comment by ayshahussain141** [comment]
Screenshot 2022-10-13 200829

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**